test/test_fake_tensor.py

```diff
@@ -778,6 +778,15 @@ def test__adaptive_avg_pool2d_backward(self):
             grad_in = torch.ops.aten._adaptive_avg_pool2d_backward(grad_out, inp)
             self.assertTrue(torch._prims_common.suggest_memory_format(grad_in) == torch.channels_last)
 
+    def test_export_numpy(self):
+        class MyNumpyModel(torch.nn.Module):
+            def forward(self, input):
+                return input.numpy()
```

Comment:
Can you add an operator which is only defined on numpy but not pytorch?

Reply:
is there such an op? :)